src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction.java

```diff
@@ -568,18 +571,76 @@ private synchronized void load() {
             LOGGER.log(Level.SEVERE, "Failed to load the configuration from config path = "+config,e);
         }
     }
-    
+
+    /**
+     * Represents the different response formats (JSON, pretty JSON, XML).
+     */
+    private enum ResponseFormat {
+        JSON("json", "application/json;charset=UTF-8", Boolean.FALSE),
+        PRETTYJSON("json-pretty", "application/json;charset=UTF-8", Boolean.TRUE),
+        XML("xml", "text/xml;charset=UTF-8", Boolean.FALSE);
+
+        private String alias;
+
+        private String contentType;
+        
+        private Boolean pretty;
+
+        private static final ResponseFormat DEFAULT = JSON;
+
+        private ResponseFormat(final String alias, final String contentType, final Boolean pretty) {
+            this.alias = alias;
+            this.contentType = contentType;
+            this.pretty = pretty;
+        }
+
+        public static ResponseFormat fromAlias(final String alias) {
+            if (alias == null) {
+                return DEFAULT;
+            }
+            if (alias.equals("json")) {
```

Comment:
@rsandell If you use `equalsIgnoreCase` these URLs will be the same:

1. `http://localhost:8080/jenkins/docker-traceability/rawContainerInfo?format=json-pretty&id=...`
1. `http://localhost:8080/jenkins/docker-traceability/rawContainerInfo?format=JSON-PRETTY&id=... `

But according to the [specification] (http://tools.ietf.org/html/draft-ietf-httpbis-p1-messaging-25#page-19) *QueryParams* are case-sensitive:

> The scheme and host are case-insensitive and normally provided in lowercase; all other components are compared in a case-sensitive manner.

IMHO, I prefer to use `equals` in this case.